internal/generator/model/relation.go

```diff
@@ -26,11 +26,12 @@ import (
 
 // StandaloneRelation in a model
 type StandaloneRelation struct {
-	Id         IdUid   `json:"id"`
-	Name       string  `json:"name"`
-	Target     *Entity `json:"-"`
-	TargetId   IdUid   `json:"targetId"`
-	UidRequest bool    `json:"-"` // TODO
+	Id         IdUid                  `json:"id"`
+	Name       string                 `json:"name"`
+	Target     *Entity                `json:"-"` // TODO consider changing to TargetName
+	TargetId   IdUid                  `json:"targetId"`
+	UidRequest bool                   `json:"-"` // TODO test
```

Comment:
> what kind of test? 

Same like for [rename-property](https://github.com/objectbox/objectbox-generator/tree/main/test/comparison/testdata/fbs/rename-property).

> does this take a long time?  

it's pretty quick to do with the "comparison tests", quite a bit less so with the manual integration tests

Reply:
If you mind the comparison tests so much, we can only do the negative test case (*.fail.fbs) to check the UID request works fine. Therefore, there would be no generated code.

Reply:
the usage is clear and explained with comments when you see it in the context ("find usage")

Reply:
I've added such comment two days ago so I hope that helps

Reply:
no, not at all, it doesn't say that. This really isn't such a big topic, just look for usage in the IDE and it's clear what it does and how it does it (basically step 1 and 2 here: https://docs.objectbox.io/advanced/data-model-updates#how-to-and-example). You know the same could be applied to anything, like "what is TargetId field" for example. It's not really feasible to have 100% documentation everywhere you could encounter something. I have to assume people use editors and don't only look at code in GH PRs :D

Reply:
no, `Id`, as in property "number" is always zero in the model coming from the FBS parser - It can't be anything else just because the FBS doesn't have any `id=value` annotation (there's the `id` annotation but that's a different thing, marking the property as the object ID).

And the code you've pasted here doesn't imply that either, it actually only says that ID is empty if UID annotation is given and it's not empty. Doesn't say anything about other cases, if there isn't any UID annotation or if the UID annotation is empty
```go
		if value.Details["uid"] != nil {
			if len(value.Details["uid"].Value) == 0 {
				// in case the user doesn't provide `objectbox:"uid"` value, it's considered in-process of setting up UID
				// this flag is handled by the merge mechanism and prints the UID of the already existing entity
				relation.UidRequest = true
			} else if uid, err := strconv.ParseUint(value.Details["uid"].Value, 10, 64); err != nil {
				return fmt.Errorf("can't parse uid on relation %s - %s", relation.Name, err)
			} else {
				relation.Id = model.CreateIdUid(0, uid)
			}
		}
```

In other words `Id` and `UidRequest` are independent, you can't say whether user has requested UID "error" just because there's no `Id` in the source model